pkg/apis/servicebinding/v1alpha3/servicebinding_lifecycle.go

```diff
@@ -19,12 +19,13 @@ const (
 	ServiceBindingConditionReady            = "Ready"
 	ServiceBindingConditionServiceAvailable = "ServiceAvailable"
 	ServiceBindingConditionProjectionReady  = "ProjectionReady"
+	InitializeConditionReason               = "Unknown"
 )
 
-func (bs *ServiceBindingStatus) InitializeConditions() {
-	ready := metav1.Condition{Type: ServiceBindingConditionReady}
-	serviceAvailable := metav1.Condition{Type: ServiceBindingConditionServiceAvailable}
-	projectionReady := metav1.Condition{Type: ServiceBindingConditionProjectionReady}
+func (bs *ServiceBindingStatus) InitializeConditions(now metav1.Time) {
+	ready := metav1.Condition{Type: ServiceBindingConditionReady, Status: metav1.ConditionUnknown, LastTransitionTime: now, Reason: InitializeConditionReason}
```

Comment:
Wondering if it would be simpler to set the `metav1.time` inside the method. Perhaps the unit test will be challenging!

Reply:
Yup you guessed it right